lib/pure/random.nim

```diff
@@ -172,32 +167,28 @@ proc skipRandomNumbers*(s: var Rand) =
   ## generated in each thread will never overlap as long as no thread generates
   ## more than 2^64 random numbers.
   ##
-  ## The following example below demonstrates this pattern:
-  ##
-  ## .. code-block::
-  ##   # Compile this example with --threads:on
-  ##   import random
-  ##   import threadpool
-  ##
-  ##   const spawns = 4
-  ##   const numbers = 100000
-  ##
-  ##   proc randomSum(rand: Rand): int =
-  ##     var r = rand
-  ##     for i in 1..numbers:
-  ##       result += rand(1..10)
-  ##
-  ##   var r = initRand(2019)
-  ##   var vals: array[spawns, FlowVar[int]]
-  ##   for val in vals.mitems:
-  ##     val = spawn(randomSum(r))
-  ##     r.skipRandomNumbers()
-  ##
-  ##   for val in vals:
-  ##     echo ^val
-  ##
-  ## See also:
+  ## **See also:**
   ## * `next proc<#next,Rand>`_
+  runnableExamples("--threads:on"):
+    import std/[random, threadpool]
+
+    const spawns = 4
+    const numbers = 100000
+
+    proc randomSum(rand: Rand): int =
+      var r = rand
+      for i in 1..numbers:
+        result += rand(1..10)
+
+    var r = initRand(2019)
+    var vals: array[spawns, FlowVar[int]]
+    for val in vals.mitems:
+      val = spawn randomSum(r)
+      r.skipRandomNumbers()
+
+    for val in vals:
+      echo ^val
```

Comment:
```suggestion
      assert abs(^val - numbers * 5) / numbers < 0.1
```

Reply:
This doesn't pass, I'd just check `^val in numbers..(numbers * 10)`.

Reply:
sorry i had also changed `result += rand(0..10)` to `result += rand(0..10)` to make it pass; it's an intuitive test, showing no statistical bias. Better than the weaker `assert abs(^val - numbers * 5) / numbers < 0.1`

Reply:
So you mean I should use `result += rand(0..10)` or rather my version?

Reply:
yes; just makes the numbers more intuitive